josue/francisco-copy/js/app.test.js

```diff
@@ -10,46 +10,47 @@ const App = require('./app-utils.js');
 
 let dom;
 let document;
-let AppElements;
+let BUTTONS;
+let SECTIONS;
+let FORMS;
 
 describe('index.html', () => {
   beforeEach(() => {
     dom = new JSDOM(html, { runScripts: 'dangerously' });
     document = dom.window.document;
     // Give context to our App Buttons
     App.setContext(document);
-    AppElements = App.createAppElements();
+    ({ BUTTONS: BUTTONS, SECTIONS: SECTIONS, FORMS: FORMS } = App.createAppElements());
```

Comment:
What is this line for? Shouldn't it be `({ BUTTONS: BUTTONS, SECTIONS: SECTIONS, FORMS: FORMS }) = App.createAppElements();` (Instead of enclosing parenthesis at the end of the assignation we do it before the `=` sign)